utils/luci-mod-freifunk-ui/luasrc/model/cbi/freifunk/basics.lua

```diff
@@ -15,11 +15,11 @@ community.rmempty = false
 
 local profile
 for profile in fs.glob(profiles) do
-	local name = uci:get_first(profile, "community", "name") or "?"
-	community:value(string.gsub(profile, "/etc/config/profile_", ""), name)
+	local n = string.gsub(profile, "/etc/config/profile_", "")
+	local name = uci:get_first("profile_"..n, "community", "name") or "?"
+	community:value(n, name)
 end
 
-
```

Comment:
As the whole change of the code seems exactly the same as upstream, I prefer not to remove this empty line, as long as it's also in the upstream code, to keep both versions easily comparable.